commons/agents.py

```diff
@@ -112,13 +140,79 @@ async def run(self, image: BinaryContent, user: str = "ANON", prompt: str = None
         return response.data.rate
 
 
+class ReasonerMemeRater:
```

Comment:
If we always use this agent can we get rid of the KittyMemeRater?

Reply:
It depends, do we want to use the standard direct one or the reasoner one, after testing we can get rid of one of them, the new one gives a description of the meme and then the reasoner gives a rating based on the description, the previous one just uses the image directly which is why I didn't put the reasoner one as a default.

Reply:
I've been testing out the reasoner's ratings, it seems much more strict but reasonably fair. Maybe we'll leave the code as is for now but I'll keep it enabled in conf until someone complains.